### ISSUE TITLE
incomplete readline_ in windows binaries

### ISSUE THREAD

**Issue by berbeflo** [php/doc-en#1482]
I realized that the windows binaries (e.g. `PHP 8.1.0 (cli) (built: Nov 23 2021 21:46:10) (NTS Visual C++ 2019 x64)`, downloaded from the official website do not contain all readline_ functions that are explained in the documentation. 
It seems, the following are missing:
- readline_callback_handler_install
- readline_callback_handler_remove
- readline_callback_read_char
- readline_on_new_line
- readline_redisplay

I'm not sure if there is an issue with the compilated binary. Or if those functions are not available on windows. If so, this should be stated in the docs.

**Comment by cmb69** [comment]
Yeah, [several readline functions are only conditionally defined](https://github.com/php/php-src/blob/PHP-8.0/ext/readline/readline.stub.php), i.e. they depend on features provided by the readline/editline library being used. [wineditline](https://github.com/winlibs/wineditline) does neither support `HAVE_RL_CALLBACK_READ_CHAR` nor `HAVE_RL_ON_NEW_LINE`.
